lib/src/main/java/uk/gov/di/ipv/cri/kbv/api/gateway/KeyStoreLoader.java

```diff
@@ -12,36 +12,31 @@
 import java.util.Objects;
 import java.util.UUID;
 
-class KeyStoreLoader {
+public class KeyStoreLoader {
     private static final Logger LOGGER = LoggerFactory.getLogger(KeyStoreLoader.class);
     private final ConfigurationService configurationService;
 
-    KeyStoreLoader(ConfigurationService configurationService) {
+    public KeyStoreLoader(ConfigurationService configurationService) {
         this.configurationService =
                 Objects.requireNonNull(
                         configurationService, "configurationService must not be null");
     }
 
-    private String getKeyStorePath() {
-        try {
-            File file = Files.createTempFile(UUID.randomUUID().toString(), ".tmp").toFile();
-            Path tempFile = file.toPath();
-            Files.write(
-                    tempFile,
-                    Base64.getDecoder()
-                            .decode(configurationService.getSecretValue("experian/keystore")));
-            return tempFile.toString();
-        } catch (IllegalArgumentException | NullPointerException | IOException e) {
-            LOGGER.error("Persist keystore to file failed", e);
-            return null;
-        }
+    private String getKeyStorePath() throws IOException {
+        File file = Files.createTempFile(UUID.randomUUID().toString(), ".tmp").toFile();
+        Path tempFile = file.toPath();
+        Files.write(
+                tempFile,
+                Base64.getDecoder()
+                        .decode(configurationService.getSecretValue("experian/keystore")));
+        return tempFile.toString();
     }
 
     private String getPassword() {
         return this.configurationService.getSecretValue("experian/keystore-password");
     }
 
-    void load() {
+    void load() throws IOException {
         System.setProperty("javax.net.ssl.keyStoreType", "pkcs12");
         System.setProperty("javax.net.ssl.keyStore", getKeyStorePath());
```

Comment:
Can we catch-log-throw on the getKeyStorePath so that we know what the error is? Help's with debugging in the future as we might lose context of the error.